ui/src/components/table/CustomTableRow.jsx

```diff
@@ -73,6 +74,21 @@ function CustomTableRow(props) {
                             />
                         </Tooltip>
                     )}
+                    {rowActions.includes('search') && (
+                        <Tooltip
+                            content={_(
+                                `Go to search for events associated with ${selectedRow.name}`
+                            )}
+                        >
+                            <ActionButtonComponent
+                                appearance="flat"
+                                icon={<Magnifier screenReaderText={null} size={1} />}
+                                to={`/app/search/search?q=search%20index%3D_internal%20source%3D*${selectedRow.name}*`}
+                                className="searchBtn"
+                                inline
```

Comment:
```suggestion
                                inline={false}
```

Reply:
we want inline to be true, as it will break the css for the other three buttons.

Reply:
sorry, done what?